darwin/importer/formats/nifti.py

```diff
@@ -0,0 +1,288 @@
+import json
+import warnings
+import zipfile
+from collections import OrderedDict, defaultdict
+from pathlib import Path
+from typing import Dict, List, Optional, Sequence, Union
+
+try:
+    import cc3d
+    import nibabel as nib
+except ImportError:
+    import_fail_string = """
+    You must install `darwin-py` with `pip install darwin-py[medical]`
+    in order to import with using nifti format
+    """
+    print(import_fail_string)
+import numpy as np
+from jsonschema import validate
+from upolygon import find_contours
+
+import darwin.datatypes as dt
+from darwin.importer.formats.nifti_schemas import nifti_import_schema
+from darwin.version import __version__
+
+
+def parse_path(path: Path) -> Optional[List[dt.AnnotationFile]]:
+    """
+    Parses the given ``nifti`` file and returns a ``List[dt.AnnotationFile]`` with the parsed
+    information.
+
+    Parameters
+    ----------
+    path : Path
+        The ``Path`` to the ``nifti`` file.
+
+    Returns
+    -------
+    Optional[List[dt.AnnotationFile]]
+        Returns ``None`` if the given file is not in ``json`` format, or ``List[dt.AnnotationFile]``
+        otherwise.
+    """
+    if not isinstance(path, Path):
+        path = Path(path)
+    if path.suffix != ".json":
+        return None
+    with open(path, "r") as f:
+        data = json.load(f)
+        validate(data, schema=nifti_import_schema)
+    nifti_annotations = data.get("data")
+    if nifti_annotations is None:
+        return None
+    annotation_files = []
+    for nifti_annotation in nifti_annotations:
```

Comment:
Are those annotations all for different items? Each `annotation_file` is imported separately and running clear of annotations by default for that item. So having more then one `annotation_file` per Darwin Item would lead to some unexpected behaviors.

Reply:
this is each for a separate item 👍  so should be ok